Add docker example (#40)
atlas-engine/AtlasEngine.FullstackServer: docker-example/README.md

# AtlasEngine Docker

This demonstrates how to configure and start an AtlasEngine with a Postgres database via Docker.

## How Do I Set This Project Up?

### Prerequesites

* Docker
* Docker Compose

### Setup/Installation

```bash
# run in the directory of this README
docker-compose pull
```

## How do I use this project?

### Usage

```bash
# run in the directory of this README
docker-compose up

> ```

Reviewer: Würd ich hier vllt. unter einer Headline zusammenfassen, weil hier ja auch nicht viel mehr als der Befehl steht.

Evtl. so:

```md
## How to use

### Prerequesites

* Docker
* Docker Compose

### Installation & Usage

Run the following commands in the directory of this README:

To setup and install the docker container:

`
docker-compose pull
`

To start the docker container: 

`
docker-compose up
`
```

Kann ich hier wegen der Snippets leider nicht als suggestion machen. 

Ist aber auch nur als optionale Anregung gedacht. :+1:

> Now there is an [AtlasEngine server](https://github.com/atlas-engine/AtlasEngine.FullstackServer) running, using the config in `atlas-engine/config/development`, using `atlas-engine/config` for storage (e.g. log files).

Reviewer: Wenn ich die Dockerfile unten richtig interpretiere, müsste der Storage Pfad eigentlich `atlas-engine/storage` sein, oder?

```suggestion
Now there is an [AtlasEngine server](https://github.com/atlas-engine/AtlasEngine.FullstackServer) running, using `atlas-engine/config/development` for configuration and `atlas-engine/storage` as workspace (e.g. for log files).
```

> Check out [`docker-compose.yml`](docker-compose.yml) for how this is wired up.

Reviewer: ```suggestion
Check out [`docker-compose.yml`](docker-compose.yml) to see how this is wired up.
```
